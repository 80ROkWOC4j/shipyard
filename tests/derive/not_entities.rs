use shipyard::*;

#[system(Test)]
fn run(_: Not<Entities>) {}