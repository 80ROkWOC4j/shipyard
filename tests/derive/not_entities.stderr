error: custom attribute panicked
 --> $DIR/not_entities.rs:3:1
  |
3 | #[system(Test)]
  | ^^^^^^^^^^^^^^^
  |
  = help: message: Not will only work with component storages refered by &T or &mut T.
